osu.Game/Screens/Ranking/Statistics/StatisticItem.cs

```diff
@@ -18,7 +19,7 @@ public class StatisticItem
         /// <summary>
         /// The name of this item.
         /// </summary>
-        public readonly string Name;
+        public readonly LocalisableString? Name;
```

Comment:
Does this need to be nullable? Shouldn't a user just be providing `string.Empty` as before rather than null?

Reply:
Yeah, I was going to suggest this as well. I think there's a difference between `default(LocalisableString)` and `(LocalisableString)string.Empty`, though, and we don't provide a `LocalisableString.IsNullOrEmpty()` or similar method.

I would agree with this being non-null, but I think we need that method to do so.

Reply:
For sure. I think a method like this should be added here or framework before this change is made.

Reply:
Would it be fine if I were to add the `LocalisableString.IsNullOrEmpty()` within osu! then later on move it to the framework side of things?

Reply:
I'm already in the process of add it. I think this change can wait on it.